src/modules/Microsoft.Resources/deploymentScripts/deploy.bicep

```diff
@@ -0,0 +1,139 @@
+@description('Required. Display name of the script to be run.')
```

Comment:
Is this custom or did it come from CARML?

Reply:
AQT